packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -982,51 +1072,56 @@ class EnumMember implements PythonBase {
     this.value = value;
   }
 
+  public dependsOnModules() {
+    return new Set<string>();
+  }
+
   public emit(code: CodeMaker, _resolver: TypeResolver) {
     code.line(`${this.pythonName} = "${this.value}"`);
     this.generator.emitDocString(code, this.docs, { documentableItem: `enum-${this.pythonName}` });
   }
 }
 
-class Namespace extends BasePythonClassType {
-  protected getClassParams(_resolver: TypeResolver): string[] {
-    return [];
-  }
-}
-
 interface ModuleOpts {
   assembly: spec.Assembly;
   assemblyFilename: string;
   loadAssembly: boolean;
   package?: Package;
 }
 
-class Module implements PythonType {
-
-  public readonly pythonName: string;
-  public readonly fqn: string | null;
-
+class PythonModule implements PythonType {
   private readonly assembly: spec.Assembly;
   private readonly assemblyFilename: string;
   private readonly loadAssembly: boolean;
-  private readonly members: PythonBase[];
+  private readonly members = new Array<PythonBase>();
   private readonly package?: Package;
 
-  public constructor(name: string, fqn: string | null, opts: ModuleOpts) {
-    this.pythonName = name;
-    this.fqn = fqn;
-
+  public constructor(
+    public readonly pythonName: string,
+    public readonly fqn: string | null,
+    opts: ModuleOpts
+  ) {
     this.assembly = opts.assembly;
     this.assemblyFilename = opts.assemblyFilename;
     this.loadAssembly = opts.loadAssembly;
     this.package = opts.package;
-    this.members = [];
   }
 
   public addMember(member: PythonBase) {
     this.members.push(member);
   }
 
+  public dependsOnModules(resolver: TypeResolver) {
+    resolver = this.fqn ? resolver.bind(this.fqn, this.pythonName) : resolver;
```

Comment:
this.boundResolver?

Reply:
That's not a thing on this particular type, due to peculiarities in the inheritance chain here.